src/io/flutter/view/PerfFPSTab.java

```diff
@@ -95,11 +96,16 @@ private void buildUI() {
   }
 
   private void openInDevTools() {
+    final DevToolsManager devToolsManager = DevToolsManager.getInstance(app.getProject());
+
     // open the timeline view
-    DevToolsManager.getInstance(app.getProject()).openBrowserAndConnect(
-      app.getConnector().getBrowserUrl(),
-      "timeline"
-    );
+    if (devToolsManager.hasInstalledDevTools()) {
+      devToolsManager.openBrowserAndConnect(app.getConnector().getBrowserUrl(), "timeline");
```

Comment:
perhaps put this in a method inside `DevToolsManager` to consolidate the two uses here in the memory tab below. Something like:
```
public void openToScreen(String screen) {
  openBrowserAndConnect(app.getConnector().getBrowserUrl(), screen);
}
```

or even better consolidate all the conditional logic in DevToolsManager, and just call `devToolsManager.openToScreen("timeline")` here (and below for memory)
```
public void openDevTools(String screen) {
  if (hasInstalledDevTools()) {
     openToScreen(screen);
  } 
  else {
      final CompletableFuture<Boolean> result = installDevTools();
      result.thenAccept(o -> openToScreen(screen));
  }
}

private void openToScreen(String screen) {
  openBrowserAndConnect(app.getConnector().getBrowserUrl(), screen);
}
```